### PR TITLE
Remove genesis code check

### PR DESCRIPTION
**Pull request by ouwenkg**
# Remove genesis code check

### Requirements
Temporarily remove the genesis code check. 

#### Applicable Issues
#293

### CODE REVIEW

**Comment by kaikai1024** [comment]
Add the info of applicable Issues.

**Comment by yangby-cryptape** [comment]
A option is better.

**Comment by ouwenkg** [comment]
> Provide an input parameter to control this feature is better.

Input parameters like cita version or what? Could you please make it more detailed?

**Comment by kaikai1024** [comment]
like `skip audit` ?

**Comment by yangby-cryptape** [comment]
For example:

- ```sh
  python check_genesis.py --check-code=false
  ```

- ```sh
  python check_genesis.py --check=storage,code
  ```

**Comment by kaikai1024** [comment]
I see.

**Comment by ouwenkg** [comment]
I design it again.

**Comment by kaikai1024** [comment]
> I think there no need to check genesis code in this test. we have checked the storage in contract.

Why?

**Comment by kaikai1024** [comment]
I'll do it.

**Comment by yangby-cryptape** [comment]
I do not care about whether it is required or not.

I just do not like commented code in a [version control system].
- If this piece of code is redundant, we delete them.
- If we just have to remove it temporarily, we offer an option to control it.

Anyway, you both could keep going.

[version control system]: https://en.wikipedia.org/wiki/Revision_control

File: tests/compatibility/check_genesis.py [hunk 1]
```diff
@@ -34,7 +38,7 @@ def check(old, new):
         if 'value' in old_alloc[addr]:
             continue
         # Check the code
-        if old_alloc[addr]['code'] != new_alloc[addr]['code']:
+        if check_code is True and old_alloc[addr]['code'] != new_alloc[addr]['code']:
```
Comment (kaikai1024) [inline tests/compatibility/check_genesis.py]: `if check_code and ..`

File: tests/compatibility/check_genesis.sh [hunk 2]
```diff
@@ -14,5 +14,5 @@ cd ${BINARY_DIR} \
     --super_admin "0x0000000000000000000000000000000000000000" \
     --nodes "127.0.0.1:4000" \
 && python3 ${SOURCE_DIR}/tests/compatibility/check_genesis.py \
-    --genesis test-chain/0/genesis.json \
+    --genesis test-chain/0/genesis.json --check_code false \
```
Comment (kaikai1024) [inline tests/compatibility/check_genesis.sh]: is that ok? are you try to check the value of `args.check_code`?

File: tests/compatibility/check_genesis.sh [hunk 3]
```diff
@@ -14,5 +14,5 @@ cd ${BINARY_DIR} \
     --super_admin "0x0000000000000000000000000000000000000000" \
     --nodes "127.0.0.1:4000" \
 && python3 ${SOURCE_DIR}/tests/compatibility/check_genesis.py \
-    --genesis test-chain/0/genesis.json \
+    --genesis test-chain/0/genesis.json\
```
Comment (kaikai1024) [inline tests/compatibility/check_genesis.sh]: white space

File: tests/compatibility/check_genesis.py (hunk 1 above)
Comment (ouwenkg) [reply tests/compatibility/check_genesis.py]: That's a stupid mistake i made, i remembered it.

File: tests/compatibility/check_genesis.sh (hunk 2 above)
Comment (ouwenkg) [reply tests/compatibility/check_genesis.sh]: Done!

**Pull request merged**